app/pom.xml

```diff
@@ -4,16 +4,20 @@
     <parent>
         <groupId>org.netbeans.contrib.yenta</groupId>
         <artifactId>parent</artifactId>
-        <version>1.2-SNAPSHOT</version>
+        <version>1.2</version>
     </parent>
     <artifactId>app</artifactId>
     <packaging>nbm-application</packaging>
     <name>Yenta Test App</name>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <netbeans.run.params.ide />
         <netbeans.run.params>${netbeans.run.params.ide}</netbeans.run.params>
         <all.clusters>${project.build.directory}/${brandingToken}</all.clusters>
+        <!--
+        Apparent bug in nbm-maven-plugin - fails claiming the lib JAR is not in a list
+        of modules it is visibly in; rerun with -rf :app succeeds.
```

Comment:
Hmm. Do not know what that is. But the `NbModuleSuite` tests have given me trouble elsewhere too.